client/src/app/site/pages/meetings/pages/motions/services/common/motion-permission.service/motion-permission.service.ts

```diff
@@ -87,7 +91,11 @@ export class MotionPermissionService {
                 }
                 return (
                     this.operator.hasPerms(
-                        !(this._delegationEnabled && this._forbidDelegatorSupportMotions)
+                        !(
+                            this._delegationEnabled &&
+                            this._forbidDelegatorSupportMotions &&
+                            this.operator.user.isVoteRightDelegated
+                        )
```

Comment:
At this point it may be a good idea to turn this part into its own get method, that way it doesn't have to be written out 4 times